openmc/lattice.py

```diff
@@ -1527,3 +1801,100 @@ def show_indices(num_rings):
 
         # Join the rows together and return the string.
         return '\n'.join(rows)
+    @staticmethod
+    def show_indices_ox(num_rings):
```

Comment:
I would have a function:
```Python
def show_indices(num_rings, orientation="y"):
```
Rather than two different functions. It's ok to maintain the two logic as two internal functions (`_show_indices_y(num_rings)` and `_show_indices_x(num_rings)`)